tests/test_compile.py

```diff
@@ -165,7 +176,8 @@ def a():
 @pytest.mark.skipif(
     IS_PY3,
     reason="Print statement is gone in Python 3."
-           "Test Deprecation Warming in Python 2")
+           "Test Deprecation Warning in Python 2"
+)
```

Comment:
I do not think that this is more readable. Because of the nearly empty line above the function definition the decorator does not seem to belong to the function.